Update instructor-notes.md (#1573)
carpentries/instructor-training: instructors/instructor-notes.md

- Create an event Etherpad using the [Etherpad template][etherpad-template]). (Google Doc [template](https://docs.google.com/document/d/1P_w1rgdVk4SpXvILSS-ZKz8Ujqklfujpc_zHf8D-G1A/edit?usp=sharing) may not be maintained) and a workshop website (using the [training template][training-template]). **If you find areas where the template needs to be updated,** please email suggested changes to instructor.training@carpentries.org. You are welcome to make quick fixes (e.g. links) yourself
- Send Etherpad/Google Doc and website links to [instructor.training@carpentries.org](mailto:instructor.training@carpentries.org).
- Consider forking the Instructor Training curriculum and doing all of your preparation work based on your fork so that any changes do not impact your preparation.

### One Week before the Event

- Plan logistics with co-Trainer(s)
  - Review your teaching schedule.
- Review (or set aside time closer to the event to review) the pre-assessment survey results for your workshop
- For online events:
  - Test videoconferencing set up using login credentials provided.
  - Make a copy of the [Virtual Minute Cards template][minute-cards-template] and personalize for your event.

>   - Plan how you will have trainees watch videos. If you plan to play them on zoom, familiarize yourself with audio settings and consider an ad blocking strategy.

Author: ```suggestion
  - Plan how you will have trainees watch videos. If you plan to play them on zoom, familiarize yourself with audio settings and consider an ad management strategy.
```

- For in-person events:
  - Confirm that breakout rooms are available or make an alternate plan
  - Determine who will bring paper (if needed for concept mapping) and sticky notes (for minute cards & demonstration).
  - Make sure you are prepared for the audiovisual setup in your room with the correct dongles/connectors etc.
  - Check on availability/timing of breaks, lunch, etc.
  - Familiarize yourself with the room, building, and accessibility resources as possible.

### During the event

- Take attendance. When marking trainees absent due to missed time, be sure to add notes describing what time was missed.
- Monitor the Etherpad / Google Doc for questions and responses to exercises.
- For online events, ensure that you are logged into the Zoom room with the host key so that you can mute participants and create breakout rooms as needed.

> - For self-organised training events, it may be necessary to remind trainees to fill out the application.

Author: ```suggestion
- For self-organised training events, it may be necessary to remind trainees to fill out the [application](https://amy.carpentries.org/forms/request_training/). Be sure to have trainees select "Profile Creation for Pre-approved Trainees" and enter their group code; contact instructor.training@carpentries.org if you are not sure what code to use. 
```

### Immediately after the event

- Double check that your attendance sheet is complete. For self-organized events, email instructor.training@carpentries.org to be sure we have your attendance record.

> - If you wish, you can an email to trainees thanking them for participating. 

Author: @ndporter the checkout checklist that was linked here is very outdated. At this time we feel that our pre-training email and follow-up reminder (sent ~2 months after training) are sufficient in providing all the necessary information that trainees will need.

See [the explanation here][zoom-host-key] for step-by-step instructions on how to

> claim host in Zoom Rooms using the host key.

Author: @ndporter if you'd like to add a link to the [zoom documentation in the handbook](https://docs.carpentries.org/topic_folders/communications/tools/zoom_rooms.html#zoom-rooms), you can do that. However, I think we (CT) decided our goal was to cut back on the instructional content on our end because this adds to the maintenance load for our documentation when Zoom makes changes. So I think the most sustainable choice is to link only to instructions that are hosted and maintained by Zoom.

### General tips for online training:

- **Support the lead**
  - Ways that you can support your co-teacher when they are leading are:
    - Monitor the chat in zoom/notes
    - Help post exercises + provide instructions
    - Find links or references
  - When you take turns instructing, remember to hand-off host privileges as needed, especially if anyone is stepping away.
- **Exercise management**

>   - After the first exercise, keep a list of all the participants in a plain text file on your computer so can you can easily paste it into the shared note-taking doc for exercises. This way, everyone immediately has a place to start writing.(G.W.) Tip: randomise the order of names every once in a while so it is not always the same person that is first or last in the list, and thus may get more or less attention when you discuss the responses (LN.)

Author: ```suggestion
  - After the first exercise, keep a list of all the participants in a plain text file on your computer so can you can easily paste it into the shared note-taking doc for exercises. This way, everyone immediately has a place to start writing.(GW) Tip: randomise the order of names every once in a while so it is not always the same person that is first or last in the list, and thus may get more or less attention when you discuss the responses (LN).
```
